app/util/log.ts

```diff
@@ -4,6 +4,28 @@ import env = require('./env');
 
 const envNameFormat = winston.format((info) => ({ ...info, env_name: env.harmonyClientId }));
 
+// used to allow winston to ignore particular logs
+// also used to filter express-winston logs
+export const ignorePaths = ['/admin/workflow-ui/jobs/table', '/service/work'];
+export const ignoreMessages = [
+  'timing.frontend-request',
+  'timing.backend-request',
+  'Invoking middleware',
+  'Completed middleware',
+  'Getting work for service',
+];
+export const inIgnoreList = (str, ignoreList): boolean => ignoreList.some((i) => str.includes(i));
```

Comment:
It seems like there should be some more precise way of doing this.  logger.debug vs logger.info, maybe, or proper tags?
Someone somewhere down the line is going to be very confused about log messages that should show up not showing up and jump down a debugging rabbit hole.

Reply:
I couldn't think of a way to ignore via log level, because these messages are in a class of their own (extremely useless). They somehow need to be decoupled from every other class of log messaging. Is it better to not log them to begin with (adding conditional logic to the code that executes the logger.whatever())? How would tags work? The root of the problem is that for some routes, those messages just aren't useful and create so much noise that combing through the logs becomes unbearable. Totally agree with not wanting to confuse people.. I'll keep thinking on it.